PowerEditor/src/WinControls/Preference/preference_rc.h

```diff
@@ -421,4 +421,5 @@
 	#define IDD_CUSTOMIZED_COLOR11_STATIC  (IDD_PREFERENCE_SUB_DARKMODE + 26)
 	#define IDD_CUSTOMIZED_RESET_BUTTON    (IDD_PREFERENCE_SUB_DARKMODE + 30)
 	#define IDC_DARKMODE_TONES_GB_STATIC   (IDD_PREFERENCE_SUB_DARKMODE + 35)
+	#define IDC_CHECK_DARKMODE_PLUGIN      (IDD_PREFERENCE_SUB_DARKMODE + 36)

```

Comment:
Let's remove (comment) it. We will see users' reaction about this feature to add it back, or to add an non-GUI option  in config, or even an empty xml file as sentinel for enable this option.

Reply:
I like your idea of using non-GUI option, especially empty xml.

Reply:
> Also current implementation require restart, user still must close Notepad++ to apply new config.

True. But it doesn't matter. This info will be noted in https://npp-user-manual.org/